prow/e2e-cluster_wide-auth.sh

```diff
@@ -28,44 +28,15 @@ set -u
 # Print commands
 set -x
 
-PROJECT_NAME=istio-testing
-ZONE=us-central1-f
-MACHINE_TYPE=n1-standard-4
-NUM_NODES=1
-CLUSTER_NAME=cluster-wide-auth-$(uuidgen | cut -c1-8 | tr "[A-Z]" "[a-z]")
-CLUSTER_VERSION=$(gcloud container get-server-config --project="${PROJECT_NAME}" --zone="${ZONE}" --format='value(defaultClusterVersion)')
-echo "Default cluster version: ${CLUSTER_VERSION}"
+ROOT=(bazel info workspace)
+source "${ROOT}/prow/cluster_lib.sh"
 
-CLUSTER_CREATED=false
-
-delete_cluster () {
-    if [ "${CLUSTER_CREATED}" = true ]; then
-        gcloud container clusters delete ${CLUSTER_NAME} --zone ${ZONE} --project ${PROJECT_NAME} --quiet \
-          || echo "Failed to delete cluster ${CLUSTER_NAME}"
-    fi
-}
 trap delete_cluster EXIT
+create_cluster 'cluster-wide-auth'
 
 if [ -f /home/bootstrap/.kube/config ]; then
   sudo rm /home/bootstrap/.kube/config
 fi
 
-gcloud container clusters create ${CLUSTER_NAME} --zone ${ZONE} --project ${PROJECT_NAME} --cluster-version ${CLUSTER_VERSION} \
```

Comment:
it seems we were already creating a new cluster for this test ? Sorry, I am not very familiar with these scripts. So, which part in this PR makes the change to ensure that going forward we create a new cluster when running e2e-cluster_wide-auth.sh (looks like that was the case before this PR too, I see code that does `cluster create`)

Reply:
I created a library to not have to duplicate the code. The library is imported by source "${ROOT}/prow/cluster_lib.sh"
and create_cluster creates the cluster.

Reply:
I see now. thanks